### DISCUSSION TITLE
run inventree without REDIS

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#8062] [Q&A]
I'm trying to depoy Invetree without cache and here's my seeting:
INVENTREE_CACHE_ENABLED=False
Is this sufficient?
Asking because after this the image seems to keep get Redis connetion error.

**Comment by SchrodingersGat** [comment]
@jesseyu1984918 you are correct, that should be sufficient. Can you please share the full error log you are seeing?

**Reply by jesseyu1984918** [reply]
Loading config file : /home/inventree/data/config.yaml
2024-09-03T19:10:49.392330276Z Python version 3.11.9 - /usr/local/bin/python
/root/.local/lib/python3.11/site-packages/allauth/exceptions.py:9: UserWarning: allauth.exceptions is deprecated, use allauth.core.exceptions
  warnings.warn("allauth.exceptions is deprecated, use allauth.core.exceptions")
Traceback (most recent call last):
  File "/root/.local/lib/python3.11/site-packages/django_redis/cache.py", line 29, in _decorator
    return method(self, *args, **kwargs)
2024-09-03T19:10:56.981232831Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/lib/python3.11/site-packages/django_redis/cache.py", line 99, in _get
    return self.client.get(key, default=default, version=version, client=client)
2024-09-03T19:10:56.981437578Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/lib/python3.11/site-packages/django_redis/client/default.py", line 260, in get
    raise ConnectionInterrupted(connection=client) from e
django_redis.exceptions.ConnectionInterrupted: Redis ConnectionError: Error -2 connecting to inventree-cache:6379. Name does not resolve.

**Comment by SchrodingersGat** [comment]
Odd... Can you try also setting `INVENTREE_CACHE_HOST=""`

**Reply by jesseyu1984918** [reply]
Ya...adding this works, but now it complains that global cache is enabled but no cache host is set up. Guess the INVENTREE_CACHE_ENABLED didn't work for some reason

**Reply by SchrodingersGat** [reply]
Ok, I've raised an issue for this then so we can look into it - https://github.com/inventree/InvenTree/issues/8063

**Comment by SchrodingersGat** [comment]
@jesseyu1984918 can you please provide your full version information? I cannot reproduce this on my setup.

**Comment by jesseyu1984918** [comment]
I'm using the stable release, I guess it's my environment because I have
the same variable set in another deployment and it works fine so far.
